pynopegl/init_header.py

```diff
@@ -233,11 +233,11 @@ def from_params(
         duration: Optional[float] = None,
         framerate: Optional[Tuple[int, int]] = None,
         aspect_ratio: Optional[Tuple[int, int]] = None,
-    ):
+    ) -> "Scene":
```

Comment:
pynopegl: add typing information *to* the Scene object ?

Reply:
done